.config-schema.json

```diff
@@ -78,7 +78,7 @@
             "type": "array",
             "items": {
                 "type": "string",
-                "enum": []
+                "enum": ["enable-modular-models"]
```

Comment:
The main reason I'm asking we keep it experimental is to let it be used for a week or two and gather feedback in case we need to change the API, or we missed a place where we're not properly handling 1.2